client/webserver/site/src/html/login.tmpl

```diff
@@ -8,6 +8,8 @@
         <div class="pb-2">
           <label for="pw" class="pl-1 mb-1">Password</label>
           <input type="password" class="form-control select" id="pw" autocomplete="current-password">
+          <label for="rememberPass" class="pl-1 mb-1">Remember my password</label>
+          <input type="checkbox" id="rememberPass">
```

Comment:
We'll probably want something similar on the register page too, no? Otherwise the user would have to log out and log back in to use the feature?